runtime/onert/core/src/util/shapeinf/Range.cc

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/ShapeInference.h"
+#include <cmath>
+namespace onert
+{
+namespace shape_inference
+{
+// helper function
+template <float *> ir::Shape inferRangeShape(float *start_val, float *limit_val, float *delta_val);
```

Comment:
Please remove this declaration.

Reply:
Agreed, this is unecessary, and will be removed. Thank you for spotting.